src/components/ebay-icon-button/index.marko

```diff
@@ -28,7 +29,8 @@ $ {
     class=[
         input.class,
         'icon-btn',
-        isBadged && `icon-btn--badged`
+        isBadged && `icon-btn--badged`,
+        input.transparent && 'icon-btn--transparent'
```

Comment:
So after speaking with Randy about the buttons for the breadcrumbs overflow, he agreed that the button should have some indication of interactivity - I added a background color for that. We might want to verify with Randy that the same application should be used here. For the sake of consistency, we'd likely want to carry that over to this as well. 
<img width="591" alt="image" src="https://user-images.githubusercontent.com/1675667/200652965-cd88411f-7c92-42db-b054-d7b7a5ef806b.png">

Reply:
Now that I look at it, this comment might not have been added to the most proper place in the code though it might have ramifications for this too.